src/components/ConfigSettings.jsx

```diff
@@ -128,6 +128,8 @@ export default function ConfigSettings({
           <Map
             serverSettings={serverSettings}
             params={match.params}
+            zoomControl={false}
```

Comment:
I guess I'm slightly confused why you added zoomControl and locateControl here? <Map /> component is one of ReactMap's component, not a React Leaflet Component, you can see on line 124 in the <MapContainer /> component, ZoomControl is already being disabled.